README.md

```diff
@@ -16,7 +18,7 @@ Also, check out these [resources powered by XQuery](https://github.com/joewiz/xq
 
 Some people can learn languages by reading online tutorials. Some people like to dive into a book, or have one on their shelf for reference.
 
-- [XQuery: Search across a variety of XML Data](http://shop.oreilly.com/product/9780596006341.do), Priscilla Walmsley, O'Reilly 2007. The standard text for XQuery 1.0. Also introduces [functx](http://www.xqueryfunctions.com/). Out of scope, though, is the topic of building applications with XQuery.
+- [XQuery: Search across a variety of XML Data](http://shop.oreilly.com/product/9780596006341.do), Priscilla Walmsley, O'Reilly 2007. The standard text for XQuery 1.0. Also introduces [functx](http://www.xqueryfunctions.com/). Out of scope, though, is the topic of building applications with XQuery. A [2nd edition](git@github.com:emchateau/learnxquery.git) annonced for september 2015 is on the making.
```

Comment:
Thanks. If either of these are too much trouble for you, though, I can incorporate the changes on my end.